Packs/XSOARContentUpdateNotifications/Playbooks/playbook-Content_Update_Manager.yml

```diff
@@ -2476,6 +2484,15 @@ inputs:
     \ You must have an installed and configured Slack integration. Also, make sure\
     \ the Cortex XSOAR application has access to this channel."
   playbookInputQuery:
+- key: slackchannelid
+  value:
+    complex:
+      root: incident
+      accessor: contentnotificationslackchannelid
+  required: false
+  description: "Provides a Slack username for the new content notifications. Note:\
```

Comment:
Seems like this description is referring to "username" and not a slack channel id.

Reply:
It was a typo. Fixed.